help/source/conf.py

```diff
@@ -120,7 +127,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+html_static_path = []
```

Comment:
any particular reason to remove the `_static` item from the list?

Reply:
We don't actually have any static content right now, so this was causing a warning during the build. I could have also created an empty directory to resolve this, but figured we'd worry about it later. I set up the Read the Docs build so that it would fail on warnings so we don't create any broken links.